frontend/src/components/themes/E15Theme.scss

```diff
@@ -0,0 +1,15 @@
+@import './DefaultTheme.scss';
+
+*:deep() {
+  // Override any of the default theme variables here
+
+  // Color palette
+  // -------------
+
```

Comment:
To, že tam nejsou ty primary je záměr?

Reply:
Poslali jenom červenou, tak jsem tam dal jenom secondary